control/control.yaml

```diff
@@ -51,6 +51,36 @@ components:
           type: string
           enum: [start, stop, pause]
 
+    Clear.Metrics:
+      description: >-
+        Clear metrics.
+      type: object
+      required: [metrics_choice]
+      properties:
+        port_names:
+          description: >-
+            The names of port objects to.
```

Comment:
comment is incomplete. Perhaps rephrase as "The names of port objects for which metrics will be cleared"?

Reply:
modified